source/administration/incident-response-application.rst

```diff
@@ -0,0 +1,466 @@
+  :depth: 1
+
+How Can I Try The App?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. `Sign up for the closed Alpha program <https://docs.google.com/forms/d/e/1FAIpQLSf4Rr1YnofQQnKHJuL0Cgz_DaCUitt_Atik7K9KXsDefCyXlg/viewform>`_. If you're selected to join the Alpha program, you will receive an email from us with a plugin binary.
+
+2. Upload the plugin binary in **System Console > Plugins > Plugin Management** `following these steps <https://about.mattermost.com/default-plugin-uploads>`_.
+
+3. Enable the plugin from the **Installed Plugins** list.
+
+4. Use `one of these sample app workflows <// TODO Add a link>`_ to try the app. To upload these workflows to your Mattermost server:
+
+ - save the app workflow schema to a JSON file
+ - go to any Mattermost channel and post ``/workflow edit`` to open the workflow editor
+ - click **Choose .json file** and select the JSON file containing the app workflow schema
+ - click **Upload**
+
+5. Post the trigger word defined in the workflow schmea to start the workflow!
+
+See the documentation below to get help on how to configure and manage your app workflows, or open an issue at https://forum.mattermost.org and we'd be happy to help you.
+
+Incident Response App Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The app is configured through a JSON file, which you can upload as a System Administrator via ``/workflow edit``. We highly recommend using a text editor file that automatically validates syntax errors of your JSON file.
+
+The JSON file looks something as follows.
+
+  .. code-block:: json
+
+    [
+      {
+        "name": "security_issue",
+        "triggers": [
+          {
+            "name": "issuereported",
+            "type": "text",
+            "match": "#sec"
+          }
+        ],
+        "steps": [
+          {
+            "name": "Setup",
+            "start_actions": [
+              {
+                "name": "CreateWarroom",
+                "type": "create_channel",
+                "channel_name": "security-issue-{{.Instance.Number}}",
+                "channel_displayname": "Security Issue {{.Instance.Number}}"
+              },
+              ...
+              {
+                "type": "transition_to",
+                "name": "TransitionToTriage",
+                "to": "Triage"
+              }
+            ]
+          },
+          ...
+          {
+            "name": "Resolved",
+            "start_actions": [
+              {
+                "name": "ResolveWarroom",
+                "type": "post",
+                "channel_name": "{{.Action.CreateWarroom.ChannelName}}",
+                "message": "Resolved."
+              },
+              ...
+              {
+                "name": "finished",
+                "type": "finished_workflow"
+              }
+            ]
+          }
+        ]
+      }
+    ]
+
+There are four components to an incident app workflow schema: triggers, steps, actions and transitions. Each one is explained in more detail below.
```

Comment:
Somewhere we should mention that names have to be go identifiers. Go identifiers are: "An identifier is a sequence of one or more letters and digits. The first character in an identifier must be a letter. A letter is a unicode letter or an underscore."